M/MParT/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MParT"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/MeasureTransport/MParT.git", "64e4b8ee7637382ab7f9aa091ace41ed6a5cbcc6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir MParT/build && cd MParT/build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DMPART_BUILD_TESTS=OFF \
+  -DMPART_PYTHON=OFF ..
+make -j${nprocs} install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> Sys.isunix(p) && nbits(p) == 64, supported_platforms())
```

Comment:
`Sys.isunix` isn't defined on `Platform`s, meh
```suggestion
platforms = filter!(p -> !Sys.iswindows(p) && nbits(p) == 64, supported_platforms())
```

Reply:
What's the problem with Windows BTW?

Reply:
Let me try compiling for windows! The library has bindings for python that don't work with Windows so we gave up trying to compile for it, and I just forgot it was a possibility.

Reply:
It looks like cmake is having some trouble configuring on windows. Since it's a low priority, I'll push that off until I decide to update this-- right now the library is at an early stage, and the julia bindings are more of a proof-of-concept (though being able to use them on windows would give an edge over python bindings 😄 )